docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -371,9 +366,20 @@ Add the mcp300x-adc-sensor module to your robot in 3 steps:
    }
    ```
 
+Save your config.
+
 This module allows you to get multiple readings at the same time from different channels of the ADC sensor.
 If you wire and configure another sensor, such as a temperature sensor on channel 1, you can add the sensor to the `"channel_map"` and get a reading from it.
-{{< alert title="Info" color="info" >}} If you would like to see how the module works, you can find its code on [GitHub](https://github.com/viam-labs/mcp300x-adc-sensor). {{< /alert >}}
+
+{{< alert title="Info" color="info" >}}
+If you would like to see how the module works, you can find its code on [GitHub](https://github.com/viam-labs/mcp300x-adc-sensor).
+{{< /alert >}}
+
+{{% alert title="Tip" color="tip" %}}
+
+Now that you have set up your robot and are able to control your motor, you can put the suction tube of your pump into the water cup, and the output tube into the plant!
```

Comment:
This needed to be moved as it was in a weird place, before finishing setting up the robot w/ the module

Reply:
I think that's a good call but I think we can put this without an alert (it's visually not nice to have multiple alerts next to each other)

Reply:
Ok, sounds good removing